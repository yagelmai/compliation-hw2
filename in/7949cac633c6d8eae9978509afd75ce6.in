
void main()
{


    if (a){
        if(x){
           print(22);
        }
    
        else{
            s2=s;
        }
    }

}
