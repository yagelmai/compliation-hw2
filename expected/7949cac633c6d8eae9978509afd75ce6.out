6: RetType ->  VOID
7: Formals -> epsilon
35: Exp -> ID
35: Exp -> ID
37: Exp -> NUM
28: ExpList -> Exp
26: Call -> ID LPAREN ExpList RPAREN
18: Statement -> Call SC
12: Statements -> Statement
14: Statement -> LBRACE Statements RBRACE
35: Exp -> ID
17: Statement -> ID ASSIGN Exp SC
12: Statements -> Statement
14: Statement -> LBRACE Statements RBRACE
22: Statement -> IF LPAREN Exp RPAREN Statement ELSE Statement
12: Statements -> Statement
14: Statement -> LBRACE Statements RBRACE
21: Statement -> IF LPAREN Exp RPAREN Statement
12: Statements -> Statement
4: FuncDecl -> RetType ID LPAREN Formals RPAREN LBRACE Statements RBRACE
2: Funcs -> epsilon
3: Funcs -> FuncDecl Funcs
1: Program -> Funcs
